runtime/rlog/rlog.go

```diff
@@ -242,6 +252,10 @@ func addContext(ctx zerolog.Context, key string, val interface{}) zerolog.Contex
 	}
 }
 
+func reserved(key string) bool {
+	return strings.HasPrefix(key, metrics.EncoreMetricKeyPrefix)
```

Comment:
This constant is logging-related, not metrics related. It would make more sense to define it in this package (but it needs to be tagged with `//publicapigen:drop` so it doesn't get added to the public API).